README.rst

```diff
@@ -25,6 +25,27 @@ Gleichzeitig ist dieses Repository der Ort, um Fragen und Erweiterungen des BO4E
 
 🇬🇧 This is a Python library that implements Business Objects for Energy `BO4E <https://www.bo4e.de/>`_.
 
+Grundsätzliche Überlegungen
+===========================
+
+Pflichtfelder (nicht nullable Attributes)
+-----------------------------------------
+Der BO4E Standard soll (in Zukunft, Stand 2023-10-05) keine fachlichen Pflichtfelder mehr enthalten.
+Die Entscheidung, was in welchen Fällen ein Pflichtfeld ist, obliegt den Anwendungen, die den Standard nutzen.
+Entsprechend ist auch die Validierung der Pflichtfelder nicht Teil des BO4E Standards sondern anwendungsspezifisch.
+Davon ausgenommen sind technische Pflichtfelder.
```

Comment:
Haben wir auch keine mehr.

Reply:
keine python-spezifischen aber generell ja schon den typ und die version.

Reply:
https://github.com/bo4e/BO4E-python/pull/613/commits/170e4fdc079e05b2f25aa35a9167f861a432681b

Reply:
Dachte wir hatten am Ende alle Attribute, auch die der ÜberBOs und ÜberCOMs optional gemacht, inklusive _version und _typ

Reply:
So haben wir es auch im Protokoll festgehalten

Reply:
IIRC hatte @snsttr gemeint, dass der Type und die Version bei ihnen mindestens technisch notwendig sind, dass wenn man z.b. aus einem Eventbus ein BO ohne Kontext bekommt, dieses überhaupt deserialisierbar ist. Auch wir brauchen den typ fürs TypeName-Handling mindestens an den Stellen, an denen wir Boneycombs verarbeiten.

Reply:
Ich bin mir ziemlich sicher, dass wir am Ende alles optional gemacht hatten, mit dem Vermerk, dass man es ja trotzdem nutzen kann, wenn man es braucht. Sonst müssen wir das Thema nochmal aufmachen.

Reply:
Wir hatten auf jeden Fall festgehalten, auch diese Felder optional zu machen. Dass die Felder für uns z.B. notwendig sind -> anwendungsspezifisch :)
Beispiel 1: Ein Endpunkt gibt dir nen riesigen Datensatz an BOs und COMs zurück, du willst aber nicht überall die `_version` mitziehen, sondern sagst, die taucht einmal im übergeordneten Objekt auf, das reicht.
Beispiel 2: Ich habe eine Datenbank, in der der Objekttyp durch die Tabelle definiert wird, eine extra Spalte dafür wäre also überflüssig.

Du siehst, es gibt Vorteile, wenn wir auch diese technischen Felder der Anwendung überlassen.

Reply:
https://github.com/bo4e/BO4E-python/pull/613/commits/8b5f21afd8cd8b272e0314503897ba79998b6fbe